lm75bd/lm75bd.c

```diff
@@ -26,8 +26,27 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  uint8_t registerPointer = 0x00;
```

Comment:
You don't need this variable you can just use the buffer that is on the line below this one